src/viam/rpc/dial.py

```diff
@@ -191,7 +191,7 @@ def __init__(self) -> None:
         self._lib.init_rust_runtime.argtypes = ()
         self._lib.init_rust_runtime.restype = ctypes.c_void_p
 
-        self._lib.dial.argtypes = (ctypes.c_char_p, ctypes.c_char_p, ctypes.c_bool, ctypes.c_void_p)
+        self._lib.dial.argtypes = (ctypes.c_char_p, ctypes.c_char_p, ctypes.c_char_p, ctypes.c_bool, ctypes.c_void_p)
```

Comment:
I think this line is too long and it's giving us a linter error. You can run `make lint` to confirm :)